test/utils.py

```diff
@@ -725,7 +736,13 @@ def apgctl(self, command, **kwargs):
 
     def start(self):
         try:
-            self.pgctl(f'-o "-p {self.port}" -l {self.log_path} start')
+            print("\n\nPG_LOG\n")
```

Comment:
This added `print` seems accidental

Reply:
Yep good catch ccdb6d8c85c09cd7e8a530558ac924b8a0282400